src/components/Cart/CartItems.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
+import "./Cart.css";
+export const CartItems = () => {
+  const { cartState, cartDispatch } = useCart();
+  const { wishlistState, wishlistDispatch } = useWishlist();
+
+  return (
+    <>
+      <div className="cart-product-container">
+        {cartState.map((product) => {
+          return (
+            <div className="cart-card" key={product._id}>
+              <div className="cart-img-container">
+                <img
+                  src={product.img}
+                  alt={product.title}
+                  className="cart-img"
+                />
+                <i
+                  className="fas fa-trash-alt delete"
+                  onClick={() =>
+                    cartDispatch({
```

Comment:
instead of writing multiple dispatch functions.

Make a common function, under that call dispatch function. Send args to that function whichever type or payload you want to send.

Example :

const xxxHandler= (typeOfDispatch, typeOfAction) => {
dataDispatch({
type: typeOfDispatch,
payload: typeOfAction,
});
};